From 0636358bc75e9d5187515ee99d7c8d490c56bd52 Mon Sep 17 00:00:00 2001
From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Date: Sat, 15 Jul 2017 00:41:49 +0300
Subject: [PATCH] arm64: dts: renesas: ulcb: enlarge cma region

Enlarge cma region since ADSP is not used on ulcb

Signed-off-by: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts | 2 +-
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts     | 2 +-
 arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts     | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
index 1ba7a22..9d83c6c 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
@@ -63,7 +63,7 @@
 		linux,cma {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x58000000 0x0 0x18000000>;
+			reg = <0x00000000 0x57000000 0x0 0x19000000>;
 			linux,cma-default;
 		};
 
diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
index c237888..7406534 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
@@ -63,7 +63,7 @@
 		linux,cma {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x58000000 0x0 0x18000000>;
+			reg = <0x00000000 0x57000000 0x0 0x19000000>;
 			linux,cma-default;
 		};
 
diff --git a/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
index b3ecbd3..9aa4292 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
@@ -53,7 +53,7 @@
 		linux,cma {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x58000000 0x0 0x18000000>;
+			reg = <0x00000000 0x57000000 0x0 0x19000000>;
 			linux,cma-default;
 		};
 
-- 
1.9.1

